NGit.Test/NGit.Patch/PatchTest.cs

```diff
@@ -80,15 +80,15 @@ public virtual void TestParse_ConfigCaseInsensitive()
 		{
 			NGit.Patch.Patch p = ParseTestPatchFile();
 			NUnit.Framework.Assert.AreEqual(2, p.GetFiles().Count);
-			NUnit.Framework.Assert.IsTrue(p.GetErrors().IsEmpty());
+			NUnit.Framework.Assert.That(p.GetErrors(), Is.Empty);
 			FileHeader fRepositoryConfigTest = p.GetFiles()[0];
 			FileHeader fRepositoryConfig = p.GetFiles()[1];
 			NUnit.Framework.Assert.AreEqual("org.eclipse.jgit.test/tst/org/spearce/jgit/lib/RepositoryConfigTest.java"
 				, fRepositoryConfigTest.GetNewPath());
 			NUnit.Framework.Assert.AreEqual("org.eclipse.jgit/src/org/spearce/jgit/lib/RepositoryConfig.java"
 				, fRepositoryConfig.GetNewPath());
-			NUnit.Framework.Assert.AreEqual(572, fRepositoryConfigTest.startOffset);
-			NUnit.Framework.Assert.AreEqual(1490, fRepositoryConfig.startOffset);
+			NUnit.Framework.Assert.AreEqual(m_Crlf ? 586 : 572, fRepositoryConfigTest.startOffset);
+			NUnit.Framework.Assert.AreEqual(m_Crlf ? 1520 : 1490, fRepositoryConfig.startOffset);
```

Comment:
We did have it assuming that all Windows Line Endings would be converted into Unix Line endings. We now don't do this as it causes issues with `\r\r\n` and instead we just use the original line endings.

So in this test the location of each `\n` will vary depending on if there are windows line endings or not.